s2sphere/sphere.py

```diff
@@ -1538,10 +1538,19 @@ def get_value(self, level):
         return math.ldexp(self.deriv(), -self.__dim * level)
 
     def get_closest_level(self, value):
-        raise NotImplementedError()
+        return self.get_min_level(
+            (math.sqrt(2) if self.__dim == 1 else 2) * value
+        )
 
     def get_min_level(self, value):
```

Comment:
Copy over the documentation while you're in here?
https://github.com/micolous/s2-geometry-library/blob/ad4673593fe23bbf8aacce4aead577db0bae9f5f/geometry/s2/s2.h#L472-L476